src/actions/IdentityAction.php

```
@@ -0,0 +1,49 @@
+
+        $request = Yii::$app->request;
+
+        if (!$request->getIsPost()) {
+            throw new MethodNotAllowedHttpException(
+                'Only POST requests are allowed.',
+            );
+        }
+
+        if (!$request->validateCsrfToken()) {
+            throw new BadRequestHttpException(
+                Yii::t('yii', 'Unable to verify your data submission.'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'beforeRun'. (undefined)

(StaticAccess)